src/lightning_app/cli/commands/app_commands.py

```diff
@@ -0,0 +1,94 @@
+import os
+import sys
+from typing import Dict, Optional
+
+import requests
+
+from lightning_app.cli.commands.connection import _resolve_command_path
+from lightning_app.utilities.cli_helpers import _retrieve_application_url_and_available_commands
+from lightning_app.utilities.commands.base import _download_command
+from lightning_app.utilities.enum import OpenAPITags
+
+
+def _run_app_command(app_name: str, app_id: Optional[str]):
+    """Execute a function in a running App from its name."""
+    # 1: Collect the url and comments from the running application
+    url, api_commands, _ = _retrieve_application_url_and_available_commands(app_id)
+    if url is None or api_commands is None:
+        raise Exception("We couldn't find any matching running App.")
+
+    if not api_commands:
+        raise Exception("This application doesn't expose any commands yet.")
+
+    full_command = "_".join(sys.argv)
+
+    has_found = False
+    for command in list(api_commands):
+        if command in full_command:
+            has_found = True
+            break
+
+    if not has_found:
```

Comment:
This could maybe be simplified:
```suggestion
    if not any(command in full_command for command in list(api_commands)):
```

Reply:
Yes, but this is meant to find the matching element, so we don't go through the for loop twice or store the matching index.